### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by cokiengchiara**
Towards https://github.com/giantswarm/giantswarm/issues/14560

This PR clarifies which apps are preinstalled vs optional depending on providers and versions.

### CODE REVIEW

File: src/content/reference/tenant-cluster-release-versions/_index.md [hunk 1]
```diff
@@ -108,11 +108,20 @@ You have several options to inspect tenant cluster release details:
...
-made optional. That means it is not installed by default when you create your
-cluster.
+Depending on your provider (AWS, Azure, or KVM), the apps NGINX IC, External DNS, and Cert Manager may be preinstalled, optional, or not available (n/a).
+
+Preinstalled apps are installed by default upon cluster creation. Optional apps can be installed from App Catalogs. In releases where they are not preinstalled, n/a apps (e.g. External DNS in certain releases) are currently not available to be installed as an optional app.
+
+| Tenant cluster release version | NGINX IC      | External DNS  | Cert Manager |
+|:------------------------------:|:-------------:|:-------------:|:------------:|
+| **AWS v10.x.x+**               | optional      | preinstalled  | preinstalled |
+| **AWS legacy**                 | preinstalled  | n/a           | optional     |
```
Comment (cokiengchiara) [inline src/content/reference/tenant-cluster-release-versions/_index.md:120]: not sure about this "legacy" terminology

Comment (glitchcrab) [reply src/content/reference/tenant-cluster-release-versions/_index.md:120]: i guess we still refer to pre-nodepool clusters as legacy

**Review by glitchcrab — APPROVED** [review]
:ok:

**Comment by cokiengchiara** [comment]
> 🆗

Just OK???? No cute GIF? 😞

**Pull request merged**